src/develop/masks/masks.c

```diff
@@ -505,126 +505,132 @@ int dt_masks_version(void)
   return DEVELOP_MASKS_VERSION;
 }
 
-int dt_masks_legacy_params(dt_develop_t *dev, void *params, const int old_version, const int new_version)
+static int dt_masks_legacy_params_v1_to_v2(dt_develop_t *dev, void *params)
```

Comment:
Unfortunately, yes. I haven't really had time to thought about the fact that there will be v2->v3, and that doing it in separate function would have been cleaner when i was in the rush of fixing flip regression (v1 >v2).